wcfsetup/install/files/lib/system/package/PackageInstallationSQLParser.class.php

```diff
@@ -260,8 +260,6 @@ protected function getIndexOwnerID($tableName, $indexName)
             return $row['packageID'];
         } elseif (isset($this->knownTables[$tableName])) {
             return $this->knownTables[$tableName];
-        } else {
-            return;
```

Comment:
This should probably rather be an explicit `return null;` to make this a total function, but one would need to carefully check all users and add real types first.